### ISSUE TITLE
Memmon not remembering locations

### ISSUE THREAD

**Issue by AL42Gaming** [relikd/Memmon#13]
I have a triple screen setup using my M1 Macbook Air screen and 2 monitors which are plugged in through a Thinkpad docking station. Whenever I plug the wire into my computer, it gets confused, throwing up the correct thing on the main screen but the apps that belong on my mac screen get put on the third screen. 

This may be due to other apps messing with Memmon, so I included a screenshot of the apps that affect most of my mac(Note: many of them have custom icons).
Additionally, to help troubleshoot I also included a screenshot of my display arrangement.
![image](https://user-images.githubusercontent.com/69176272/230761639-53c895bc-3ea2-46cd-a9f7-d30fa52b1a99.png)
![image](https://user-images.githubusercontent.com/69176272/230761647-3382292b-e227-442d-ae88-8254fad5a32f.png)

**Comment by relikd** [comment]
Yes, I stumbled upon this issue too, see https://github.com/relikd/Memmon/issues/11#issuecomment-1466864244. Unfortunatelly, I dont see an easy solution for the problem (see later comments). However, you can try and disable "Displays have separate Spaces" in System Preferences > Mission Control. If you can live without that feature, macOS will properly remember the window positions even without Memmon.

I'll keep this issue open until we have a solution for this ... though I am not sure if or when that will be.

**Comment by AL42Gaming** [comment]
This seems to have solved the issue for now, though it is very frustrating as I use my laptop in school and like to use the other windows in a different desktop. Hopefully this issue can be resolved, but thank you for the solution and for your efforts.
